test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git.
```

Comment:
- [ ] As you said in this file, it should be ignored by the `.gitignore` file. Please add this file in the `.gitignore` file as you did for the `node_modules` and remove it from this repository.